### PR TITLE
feat(widget-message-meet): Add ringtones, and call duration

### PR DESCRIPTION
**Pull request by bzang**
Most of this is in: https://github.com/ciscospark/react-ciscospark/pull/54
Only thing relevant are the last 3 commits.

### CODE REVIEW

**Comment by adamweeks** [comment]
You've got a snapshot failing on this.

File: packages/node_modules/@ciscospark/react-component-ringtone/src/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import {Component, PropTypes} from 'react';
 import {Howl} from 'howler';
 
 import RingtoneIncoming from './media/Ringtone.mp3';
-import RingtoneRingback from './media/Ringback_v7_norm.wav';
+import RingtoneRingback from './media/Ringback_v2.mp3';
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-ringtone/src/index.js]: Did you convert this or get it elsewhere?

File: packages/node_modules/@ciscospark/widget-message-meet/src/container.js [hunk 2]
```diff
@@ -178,7 +182,14 @@ export class MessageMeetWidget extends Component {
                 image={toUserAvatarUrl}
                 name={toUser.displayName}
                 onMenuClick={this.handleMenuClick}
-              />
+              >
+                {
+                  callStartTime &&
+                  <div className={classNames(`title-bar-call-timer`, styles.callTimer)}>
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-message-meet/src/container.js]: We should also figure out a way to make this available in the `widget-meet` so the call timer would be available for anyone just using that widget.

Comment (bzang) [inline packages/node_modules/@ciscospark/widget-message-meet/src/container.js]: yea. The timer component can be used wherever and has an internal store (which I really don't like, but makes a lot of sense here). The designs specifically don't include the duration anywhere, but I do think we should have that somewhere in `widget-meet`.

File: packages/node_modules/@ciscospark/react-component-ringtone/src/index.js (hunk 1 above)
Comment (bzang) [inline packages/node_modules/@ciscospark/react-component-ringtone/src/index.js]: got it from the web client. It's 1/5 the size of the wav so I figure it's better than delivering a 750kb file to the user.

Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-ringtone/src/index.js]: Cool, we can probably remove the `.wav` rules from webpack then.

**Pull request merged**